IntegrationTests/BraintreePayPal_IntegrationTests.swift

```diff
@@ -32,13 +35,35 @@ class BraintreePayPal_IntegrationTests: XCTestCase {
         waitForExpectations(timeout: 5)
     }
     
+    func testCheckoutFlow_withoutPayPalRequest_tokenizesPayPalAccount() {
```

Comment:
Can we move this to the unit test vs integration? Can also rename to `testCheckoutFlow_withoutPayPalRequest_returnsError`